mq/esworker_eventtask.py

```diff
@@ -222,14 +226,6 @@ def on_message(self, body, message):
             # make a json version for posting to elastic search
             jbody = json.JSONEncoder().encode(normalizedDict)
 
-            if isCEF(normalizedDict):
```

Comment:
Copied comment from previous PR:

Phrozyn: This block appears unused as it is. We have no doc_types of "cef"